packages/engine/src/worker/mod.rs

```diff
@@ -106,17 +116,20 @@ impl WorkerController {
     }
 
     fn shutdown(&mut self) -> Result<()> {
-        Ok(()) // TODO
+        // TODO
+        Ok(())
     }
 
     fn shutdown_with_error(&mut self, e: Error) -> Result<()> {
-        Err(e) // TODO
+        // TODO
+        Err(e)
     }
 
     async fn _run(&mut self) -> Result<()> {
         // TODO: Rust, JS
-        // let mut py_handle = self.py.run().await?;
         let mut js_handle = self.js.run().await?;
+        // let mut py_handle = self.py.run().await?;
```

Comment:
Perhaps it doesn't matter if these are reordered, but there might have been reasons python was put first (considering it involves starting up a new process and such)

Reply:
Fair point. As this is primarily a docs-PR I reverted the change.